src/runtime/virtcontainers/qemu.go

```diff
@@ -754,6 +758,15 @@ func (q *qemu) createPCIeTopology(qemuConfig *govmmQemu.Config, hypervisorConfig
 	// Deduce the right values for mem-reserve and pref-64-reserve memory regions
 	memSize32bit, memSize64bit := q.arch.getBARsMaxAddressableMemory()
 
+	// Fow now, pcie native hotplug is the only way for Arm to hotadd pci device.
+	if machineType == QemuVirt {
+		virtPcieRootPortNum := len(hypervisorConfig.VhostUserBlkDevices) + defaultVirtPcieRootPortNum
+		if hypervisorConfig.VirtioMem {
```

Comment:
I can have only one VirtioMem device?

Reply:
I find that setupVirtioMem is only called inside "startVM". That's there is only one virtio-mem device in kata.